src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -70,11 +70,9 @@ public class ConverterRegistry {
 		put(ModElementType.BLOCK,
 				Arrays.asList(new BlockLuminanceFixer(), new BlockBoundingBoxFixer(), new BlockLightOpacityFixer(),
 						new BlockRequiresCorrectToolConverter()));
-		put(ModElementType.PLANT, Collections.singletonList(new PlantLuminanceFixer()));

```

Comment:
Why did you remove those and how does this reordering fit into this PR?

Reply:
Remainder of converter commits. These lines happened to appear twice in this file.

Reply:
✔️ Fixed